### ISSUE TITLE
More use for Dual Wielding

### ISSUE THREAD

**Issue by Zicerite** [Zicerite/Gavania-Project#9]
My current thought is that there should be a benefit to Dual Wielding. 
Idea: Passive +10% Damage to all skills.

Open to suggestions though.

**Comment by Accappy** [comment]
I honestly think adding +10% Damage to all skills would be a little to op, especially for things such as the ultimates etc, it would be a nice idea to give dual wielding a passive but strength seems a little too op. ( I.e maybe speed? )

**Comment by Zicerite** [comment]
@Accappy Speed, as in movement speed, or attack speed?
I still haven't figured out how to do the attack speed mechanic yet.

Maybe additional movement speed based on the weapon's attack speed?
Slower weapons would likely have more stats, so in exchange for those additional stats, you can walk this much faster/slower?
Maybe... im not sure.

**Comment by Accappy** [comment]
Yeah, I was thinking about additional movement speed as a percentage, but based on the weapon's attack speed would be great

**Comment by Zicerite** [comment]
Will do.
Ill make the bonus very small for Super Slow weapons, and very good for Super Fast weapons. Ill figure out how much later in a different comment, so stay tuned!

**Comment by Zicerite** [comment]
does relate to #33

**Comment by Zicerite** [comment]
Delayed to 0.6
